.github/workflows/prb.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Pull Request Builder
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-jdk8:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK8
+      uses: actions/setup-java@v1.4.3
+      with:
+        java-version: 1.8
```

Comment:
Sad that it doesn't print the installed version build number. I believe, 1.8 has the latest version of 1.8.212. Then they switched to 8.0.x pattern. Consider using the major version format (8 and 11) as described here: https://github.com/actions/setup-java#basic

Reply:
We can also add a step that executes `java -version`

Reply:
adding a print java version step is simple enough